backend/pyproject.toml

```diff
@@ -64,6 +64,7 @@ documentation = "https://docs.useintegraflow.io/"
   pillow-avif-plugin = "^1.3.1"
   semantic-version = "^2.10.0"
   requests-hardened = "1.0.0b2"
+  google-auth-oauthlib = "^1.1.0"
```

Comment:
@AfeezGL Can you sync this lib with both `requirements.txt` and `requirements_dev.txt`?